features/buyer/requirements.feature

```diff
@@ -137,6 +137,7 @@ Scenario: Create user research participants
    And I don't see the 'Review and publish your requirements' link
 
 
+@copy-requirements
```

Comment:
do we need this tag here?

Reply:
We don't need it (or the file-upload one before), but I don't feel like more tags are a bad thing. :)